src/variables.less

```diff
@@ -3,4 +3,4 @@
 @background-green: #e5f3eb;
 @border-gray: @navLysGra;
 
-@ad-height: calc(~'100vh - 8rem');
+@ad-height: 100vh;
```

Comment:
"-8rem" var ment for å trekke fra høyden til dekoratøren slik at høyden på hele appen (dekoratør + annonsevisning) ble til skjermstørrelsen. Men en slik løsning gjorde at dekoratøren (nå med modia) tok unødvendig mye av viktig vertikal plass.

![image](https://user-images.githubusercontent.com/8887917/81288273-9c33ef00-9064-11ea-8f5d-8b3a3389cf6c.png)

Nå kan man scrolle forbi dekoratøren.